### ISSUE TITLE
"Select All" functionality

### ISSUE THREAD

**Issue by hannank12** [tzutalin/labelImg#505]
I was expecting a "Select All" functionality , it would be better if this tool can have this function to select multiple Boxes at a time and copy paste them. Please include this function.

**Comment by radusl** [comment]
I agree. It would be really useful and practical to have the ability to select multiple labels, move them all at the same time or delete
